scripts/integration-tests/utils/bump-babel-dependencies.js

```diff
@@ -0,0 +1,29 @@
+const fs = require("fs");
+const path = require("path");
+const cwd = process.cwd();
+const packageJSONPath = path.resolve(cwd, "./package.json");
+const content = JSON.parse(fs.readFileSync(packageJSONPath));
+
+let bumped = false;
+function bumpBabelDependency(dependencies) {
+  for (const dep of Object.keys(dependencies)) {
+    if (dep.startsWith("@babel/")) {
+      dependencies[dep] = "latest";
```

Comment:
Technically if a downstream project specifies `latest` for `@babel/` in the `package.json`, this approach will not work at all because `yarn` will then lock `latest` to old versions. But practically this scenario should never happen.